### ISSUE TITLE
Clear Cache under Storage Usage not working

### ISSUE THREAD

**Issue by Dantasstic** [TelegramMessenger/Telegram-iOS#1587]
### Checklist
- [X ] I am reporting an issue in existing functionality that does not work as intended
- [X ] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
Clear Entire Cache (_or individual chosen sections, it doesn't matter_) does not function anymore. 

### Expected Behavior
The current storage usage size should reset back to 0

### Actual Behavior
If you run it, it claims to have freed X amount of storage, but nothing changes on the screen, and the exact same cache amount is stated as before running it.

### Steps to Reproduce
1. Go to >Settings
2. Go to >Data and Storage
3. Go to >Storage Usage
4. Tap on >Clear Entire Cache

### Screenshots and Videos 
https://streamable.com/p3i8nv

### Environment
**Device:** iPhone 16 Pro Max (_also tested on iPhone SE 2020 and iPad Pro 12.9 3rd gen_)

**iOS version**: iOS 18.0.1 (_and iOS 17.7 and iPadOS 17.7_)

**App version:** 11.2

**Comment by andriisimon** [comment]
The same problem for iPhone `14 Pro Max` + iOS `18.0.1`

**Comment by Dantasstic** [comment]
Still an issue on the latest Telegram app update (11.2.1) released on October 17, 2024

**Comment by Dantasstic** [comment]
Still an issue on iOS 18.1

**Comment by TETOUTE** [comment]
When I download the Telegram API, as soon as I start the application, it goes black and doesn't let me do anything, does this happen to anyone else?

**Comment by Dantasstic** [comment]
You will need to create a new bug report for that issue

> When I download the Telegram API, as soon as I start the application, it goes black and doesn't let me do anything, does this happen to anyone else?

You need to create a separate bug report for that issue. Each issue needs its own report.

**Comment by DaNTaF** [comment]
Same here. iPhone 14 Pro Max 18.1 Telegram 11.3

**Comment by Dantasstic** [comment]
Still an issue on the latest Telegram app update (11.3) released on October 31, 2024

**Comment by ride2x4** [comment]
Same on 11.3.1, ios 15.7.1

**Comment by Dantasstic** [comment]
Still an issue on the latest Telegram app update (11.4) released on November 17, 2024

**Comment by VetaLioSTV** [comment]
Same problem on iOS 15.

By the way, actually this problem exists at least at three major updates and still not fixed, thats crazy... 😢

**Comment by Dantasstic** [comment]
The issue also exists on the latest iOS update 18.1.1

**Comment by Dantasstic** [comment]
Still an issue on the latest Telegram app update (11.5) released on December 4, 2024

**Comment by VetaLioSTV** [comment]
> Still an issue on the latest Telegram app update (11.5) released on December 4, 2024

i can confirm it, same for me. Its broken for a long time. 

@laktyushin @ali-fareed could you respond / check ?

**Comment by VetaLioSTV** [comment]
Still the same on latest update 11.5.2

**Comment by tahatiyan** [comment]
This is an old bug in ios and they have not been able to solve it yet.

**Comment by Dantasstic** [comment]
> This is an old bug in telegram on apple devices and they have not been able to solve it yet. I don’t know this bug is from apple or telegram!

Incorrect. It literally worked as it was intended to on iOS 17.7 with no issues before the first update I reported (11.2) it broken on.

**Comment by VetaLioSTV** [comment]
@Dantasstic do you still have it? Any updates?
